cloudflare-pages/action.yml

```diff
@@ -0,0 +1,62 @@
+name: 'Meticulous - Trigger tests for Cloudflare Pages'
+description: 'Trigger Meticulous tests for a Cloudflare Pages deployment'
+inputs:
+  github-token:
+    description: 'A GitHub Actions token (should have write permission on deployments)'
+    required: true
+  cloudflare-api-token:
+    description: 'API token for Cloudflare (should have Read permission for Cloudflare Pages)'
+    required: true
+  cloudflare-account-id:
+    description: 'Account ID for Cloudflare Pages (from the Workers & Pages section of the dashboard)'
+    required: true
+  cloudflare-project-name:
+    description: 'Name of your Cloudflare Pages project (from the Workers & Pages section of the dashboard)'
+    required: true
+  sleep-seconds:
+    description: 'Time to sleep before starting to poll deployment status (in seconds)'
+    required: false
+    default: "300"
```

Comment:
What are the API limits? Is there a risk we'll hit those if we reduce the polling / wait times? 5 minutes initial wait feels long, assuming build times are similar to Vercel, can we wait, say < 30 sec? And also poll every 10 sec or so.

Reply:
It's 1200 requests per 5 minutes. So assuming the user's token is only being used by us, polling once a minute would mean we can be waiting on 240 concurrent builds without hitting the limit. It might be safe to reduce that, but we might break in cases of tools like dependabot opening a load of PRs at once. Still, halving to 30 seconds is probably safe, I'll do that.